ws.nimble

```diff
@@ -8,13 +8,13 @@
 ## those terms.
 
 packageName = "ws"
-version = "0.1.0"
-author = "Status Research & Development GmbH"
+version     = "0.1.0"
+author      = "Status Research & Development GmbH"
 description = "WS protocol implementation"
-license = "MIT"
-skipDirs = @["examples", "test"]
+license     = "MIT"
+skipDirs    = @["examples", "tests"]
 
-requires "nim >= 1.2.6"
+requires "nim >= 1.2.0" # nimble will fail to install nim-ws if we are using 1.2.6 here
```

Comment:
Why is this?

Reply:
it is a nimble bug that has been around for long time and still unsolved.